temoa-uiuc/data_files/11_uiuc.sql

```diff
@@ -0,0 +1,1311 @@
+INSERT INTO "CostVariable" VALUES('uiuc', 2021, 'IMPGSL', 2021, 0.00222, 'M$/k_GAL', 'typical gasoline price');
+INSERT INTO "CostVariable" VALUES('uiuc', 2021, 'IMPDSL', 2021, 0.00248, 'M$/k_GAL', 'typical diesel price');
+INSERT INTO "CostVariable" VALUES('uiuc', 2021, 'IMPE85', 2021, 0.00199, 'M$/k_GAL', 'typical E85 price');
+INSERT INTO "CostVariable" VALUES('uiuc', 2021, 'IMPH2', 2021, 0.006, 'M$/metrictons', 'typical hydrogen price $6/kg');
+-- No cost variable considered for the ELCVCL because uses electricity produced in the system.
+
+
+CREATE TABLE "CostInvest" (
+	"regions"	text,
+	"tech"	text,
+	"vintage"	integer,
+	"cost_invest"	real,
+	"cost_invest_units"	text,
+	"cost_invest_notes"	text,
+	FOREIGN KEY("tech") REFERENCES "technologies"("tech"),
+	FOREIGN KEY("vintage") REFERENCES "time_periods"("t_periods"),
+	PRIMARY KEY("regions","tech","vintage")
+);
+--UIUC Data
+INSERT INTO "CostInvest" VALUES('uiuc','ABBOTT', 2021, 0.735, 'M$/MWth', 'cost of installing a natural gas unit');
+INSERT INTO "CostInvest" VALUES('uiuc','IMPSOL', 2021, 1.66, 'M$/MWe', 'solar farm 2.0 contract');
+INSERT INTO "CostInvest" VALUES('uiuc','ELECTROL', 2020, 1.03019, 'M$/MWe', 'avg investment cost for alkaline electrolysis cells');
+INSERT INTO "CostInvest" VALUES('uiuc','IMPWIND', 2021, 1.75, 'M$/MWe', 'if UIUC builds its own wind farm');
+INSERT INTO "CostInvest" VALUES('uiuc','NUCLEAR', 2021, 5.945, 'M$/MWth', 'cost of utility scale nuclear plant');
+-- Based on values from fuel-analysis/temoa-model.ipynb
+INSERT INTO "CostInvest" VALUES('uiuc','ELCVCL', 2021, 0.0229, 'M$/gge', 'cost of electric vehicle');
+INSERT INTO "CostInvest" VALUES('uiuc','H2VCL', 2021, 0.0116, 'M$/gge', 'cost of hydrogen vehicle');
+
+
+CREATE TABLE "CostFixed" (
+	"regions"	text NOT NULL,
+	"periods"	integer NOT NULL,
+	"tech"	text NOT NULL,
+	"vintage"	integer NOT NULL,
+	"cost_fixed"	real,
+	"cost_fixed_units"	text,
+	"cost_fixed_notes"	text,
+	FOREIGN KEY("tech") REFERENCES "technologies"("tech"),
+	FOREIGN KEY("periods") REFERENCES "time_periods"("t_periods"),
+	FOREIGN KEY("vintage") REFERENCES "time_periods"("t_periods"),
+	PRIMARY KEY("regions","periods","tech","vintage")
+);
+--uiuc data
+--INSERT INTO "CostFixed" VALUES('uiuc', 2021, 'ABBOTT', 2000, 0.0553, 'M$/GWh', '');
+--INSERT INTO "CostFixed" VALUES('uiuc', 2021, 'TURBINE', 2000, 0.03, 'M$/GWh', '');
+INSERT INTO "CostFixed" VALUES('uiuc', 2021, 'IMPWIND', 2020, 0.00002634, 'M$/kw', 'yearly fixed O&M cost for large scale solar from S&L report');
+INSERT INTO "CostFixed" VALUES('uiuc', 2021, 'IMPSOL', 2020, 0.00001525, 'M$/kW', 'yearly fixed O&M cost from from S&L report');
+--INSERT INTO "CostFixed" VALUES('uiuc', 2021, 'ELECTROL', 2020, 1, 'M$/kW-year', 'fixed O&M cost for');
+INSERT INTO "CostFixed" VALUES('uiuc', 2021, 'NUCLEAR', 2020, 0.00012164, 'M$/kW', 'yearly fixed O&M cost for utility scale nuclear from S\&L report');
```

Comment:
```suggestion
INSERT INTO "CostFixed" VALUES('uiuc', 2021, 'NUCLEAR', 2020, 0.00012164, 'M$/kW', 'yearly fixed O&M cost for utility scale nuclear from S&L report');
```
Simple strings only, no formatting.